src/Psalm/Internal/Cli/Refactor.php

```diff
@@ -284,7 +284,7 @@ function () use ($current_dir, $options, $vendor_dir): ?\Composer\Autoload\Class
             chdir($current_dir);
         }
 
-        $threads = isset($options['threads'])
+        $threads = isset($options['threads']) && is_numeric($options['threads'])
```

Comment:
Should we add some sort of notice if it's non-numeric?

Reply:
> However psalm's behavior has not changed

Actually it has a bit: https://3v4l.org/Xj0BS, don't you just love the way PHP handles casts?

I think this is probably fine though.